client/src/layout/menu.js

```diff
@@ -73,8 +73,8 @@ export function fetchMenu(options = {}) {
         title: _l("Analyze Data"),
         url: "",
         tooltip: _l("Analysis home view"),
```

Comment:
```suggestion
        url: "",
        tooltip: _l("Analyze Data"),
```

I have no idea if this does what I have in mind, but I don't think we need both title and tooltip?

Reply:
We don't; title element is just a holdover of how the menus are generated -- it doesn't get used if an icon is specified.  Instead of this, I will leave the tooltip as is and drop the old title out.

Reply:
I was actually suggesting to replace the old longer tooltip with "Analyze Data", so that people would recognize that the icon is the equivalent of the old button. Otherwise maybe we could just use "Home"?

Reply:
My suggestion would be "Tools and Current History" but I think we should push that to another PR since we can (and I'm sure would) bike shed these suggestions aggressively.

Reply:
I really like "Tools and Current History" -- how about I just do that now?